CautionaryAlertsApi.Tests/V1/UseCase/GetCautionaryAleryByAlertIdUseCaseTests.cs

```diff
@@ -0,0 +1,53 @@
+using AutoFixture;
+using CautionaryAlertsApi.V1.Gateways;
+using CautionaryAlertsApi.V1.UseCase;
+using Hackney.Shared.CautionaryAlerts.Infrastructure.GoogleSheets;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System;
+using FluentAssertions;
+using Hackney.Shared.CautionaryAlerts.Domain;
+
+namespace CautionaryAlertsApi.Tests.V1.UseCase
+{
+    public class GetCautionaryAleryByAlertIdUseCaseTests
+    {
+        private readonly GetCautionaryAlertByAlertIdUseCase _classUnderTest;
+        private readonly Mock<IUhGateway> _mockGateway;
+
+        private readonly Fixture _fixture = new Fixture();
+
+        public GetCautionaryAleryByAlertIdUseCaseTests()
+        {
+            _mockGateway = new Mock<IUhGateway>();
+            _classUnderTest = new GetCautionaryAlertByAlertIdUseCase(_mockGateway.Object);
+        }
+
+        [Test]
+        public void ReturnsAlertByAlertIdFromGateway()
+        {
+            // Arrange
+            var personId = Guid.NewGuid();
+            var alertId = Guid.NewGuid();
+            var mockAlerts = _fixture.Build<CautionaryAlert>()
+                                     .With(x => x.PersonId, personId)
+                                     .With(x => x.AlertId, alertId)
+                                     .Create();
+
+            _mockGateway
+                .Setup(x => x.GetCautionaryAlertByAlertId(personId, alertId))
+                .Returns(mockAlerts);
```

Comment:
The variable name is a bit misleading as it's plural 'mockAlert**s**'. _(plural)_
Based on the code, I think it should be 'mockAlert' _(singular)_